hashes/chaos_machine.py

```diff
@@ -6,7 +6,8 @@
 m = 5
 
 # Buffer Space (with Parameters Space)
-buffer_space, params_space = [], []
+buffer_space: list[int] = []
```

Comment:
```suggestion
buffer_space: List[int] = []
```
perhaps?

Reply:
This works in python 3.9, since [type hinting generics were added in standard collections](https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections), although this probably still needs to be refactored to `List[]` since this feature isn't implemented in python 3.7.

Reply:
We are switching to Python 3.9 generic types, so this is fine. Also, `mypy` supports it as well.